src/stylesheets/components/_form-button.scss

```diff
@@ -1,13 +1,17 @@
 .form-button {
   &__button--white {
```

Comment:
Could this variant be at the button component level instead since it extend the button 🤔 ?

```
.button {
  @extend .btn;
  
  &--white {
    //...
  }
}
```

Reply:
Moved the button variants to `stylesheets/base/_buttons.scss` for more generic approach. Updated in [cc6f0b2](https://github.com/rafayet-monon/surveyor/pull/13/commits/cc6f0b2c46001abbf1705e1d172dd8541675d5ef). Does this look okay?